Servers/models/ISO-42001/annexCategoryISO.model.ts

```
@@ -0,0 +1,118 @@
+import { Column, DataType, ForeignKey, Model, Table } from "sequelize-typescript";
+import { UserModel } from "../user.model";
+import { ProjectFrameworksModel } from "../projectFrameworks.model";
+import { AnnexCategoryStructISOModel } from "./annexCategoryStructISO.model";
+
+export type AnnexCategoryISO = {
+  id?: number; // auto generated by database
+  is_applicable: boolean;
+  justification_for_exclusion: string;
+  implementation_description: string;
+  evidence_links: Object[];
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)